cmd/infrastructure/ingress.yml

```diff
@@ -5,6 +5,10 @@ metadata:
   annotations:
     kubernetes.io/ingress.class: nginx
     cert-manager.io/issuer: "letsencrypt-api"
+    nginx.ingress.kubernetes.io/enable-cors: "false"
+    nginx.ingress.kubernetes.io/cors-allow-methods: "POST, GET, OPTIONS"
+    nginx.ingress.kubernetes.io/cors-allow-headers: "Origin, Content-Type, Accept, Authorization"
+    nginx.ingress.kubernetes.io/cors-allow-origin: "*"
```

Comment:
not actually needed, i dont use it but i'm going to leave it as its a option for future